app/templates/custom-elements/remote-screen.html

```diff
@@ -278,13 +280,19 @@
           const height = boundingRect.bottom - boundingRect.top;
           const relativeX = Math.min(1.0, Math.max(0.0, cursorX / width));
           const relativeY = Math.min(1.0, Math.max(0.0, cursorY / height));
-
+          const vwheel = evt.deltaY || 0;
+          const hwheel = evt.deltaX || 0;
+          if (vwheel || hwheel) {
+            evt.preventDefault();
+          }
```

Comment:
Instead of putting the check here, can we make it

```javascript
screenImg.addEventListener("wheel", (evt) => {
  evt.preventDefault();
  this.sendMouseEvent(evt);
  });